cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/declarations/EnumDeclaration.kt

```diff
@@ -35,32 +35,32 @@ import org.neo4j.ogm.annotation.Relationship
 
 class EnumDeclaration : Declaration() {
     @Relationship(value = "ENTRIES", direction = Relationship.Direction.OUTGOING)
-    var entriesPropertyEdge: List<PropertyEdge<EnumConstantDeclaration>> = ArrayList()
+    var entryEdges: List<PropertyEdge<EnumConstantDeclaration>> = ArrayList()
         private set
 
     @Relationship(value = "SUPER_TYPES", direction = Relationship.Direction.OUTGOING)
-    var superTypesPropertyEdge: List<PropertyEdge<Type>> = ArrayList()
+    var superTypeEdges: List<PropertyEdge<Type>> = ArrayList()
         private set
 
     @Relationship var superTypeDeclarations: Set<RecordDeclaration> = HashSet()
 
     @property:SubGraph("AST")
     var entries: List<EnumConstantDeclaration>
```

Comment:
```suggestion
    var entries by PropertyEdgeDelegate(EnumDeclaration::entryEdges)
```

then you can get rid of the get/setter